pkg/operator/ceph/csi/spec.go

```diff
@@ -394,11 +405,21 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 	return nil
 }
 
-func StopCSIDrivers(namespace string, clientset kubernetes.Interface) error {
-	logger.Warningf("Skipped removing the CSI driver")
+func StopCSIDrivers(namespace string, clientset kubernetes.Interface) {
+	_ = deleteCSIDriverInfo(clientset, CephFSDriverName)
```

Comment:
nit: to ignore the errors, I'd suggest no need to assign to `_`.

Reply:
Sure; I was trying to make explicit that there is an error and we're ignoring it. If that's not idiomatic I'll just roll over them.